superchain/configs/sepolia/base.toml

```diff
@@ -40,7 +40,7 @@ data_availability_type = "eth-da"
   ProxyAdminOwner = "0x0fe884546476dDd290eC46318785046ef68a0BA9"
   Guardian = "0x7a50f00e8D05b95F98fE38d8BeE366a7324dCf7E"
   Challenger = "0xDa3037Ff70Ac92CD867c683BD807e5A484857405"
-  Proposer = "0x20044a0d104E9e788A0C984A2B7eAe615afD046b"
+  Proposer = "0x037637067c1dbe6d2430616d8f54cb774daa5999"
```

Comment:
There is a ci failure related to this address. The error is thrown because this is not a checksummed address. Could you change this to the following: `0x037637067c1DbE6d2430616d8f54Cb774Daa5999`